tldraw/tldraw
# [fix] Allow interactions with embeds in readonly mode

Allow the users to interact with embeds in readonly mode. Not sure if this should apply to all embeds (cc @orangemug for thoughts on this)? For example, you can also start editing code sandbox embeds. 

One thing that doesn't feel quite right is that readonly mode defaults to the hand tool, so you always have to switch to select tool to get this working. I guess moving around is still the more common action though so 🤷 

### Test Plan

1. Create a multiplayer room with some embeds (youtube videos, spotify playlist).
2. Open the room in readonly mode.
3. Make sure you can interact with embeds. Double click / enter when selected, you should then be able to play the youtube videos.


### Release Note

- Allow the users to interact with embeds in readonly mode.
